packages/Modal/Content.tsx

```diff
@@ -51,7 +51,12 @@ export const Content = forwardRef<'div', ContentProps>(({ children, ...rest }, r
   const getStyles = (name?: string) => {
     if (name === 'Header') {
       return {
-        pb: components.includes('Body') || components.includes('Footer') ? space.lg : null,
+        // if the Modal have a Body but not a Footer || have a Footer but not a Body
```

Comment:
Hey @Leiksa,

I've just checked on the current alpha with/without Body/Footer.
Is it really necessary to create different cases? Can't see the issue

Reply:
Hello @cnairiwttj , 
We noticed a spacing problem on the modal header on wttj-front, indeed it was 16px instead of 32px.
Then, while checking the design system I noticed that the spacing was different if there was a footer or not (see the DS screenshot below). Available if you have any questions. 

<img width="333" alt="Screenshot 2023-02-14 at 17 11 04" src="https://user-images.githubusercontent.com/36013102/218793900-364e3076-b135-4249-93c4-87bf8e74bc7e.png">
<img width="362" alt="Screenshot 2023-02-14 at 17 10 57" src="https://user-images.githubusercontent.com/36013102/218793909-dc9330dd-50e2-4b5b-bb66-fd912e4290e5.png">
adding was wrong

Reply:
Ok thanks a lot for the explanation @Leiksa 
I honestly think it will be difficult to maintain but I understand the need. 
It would be nice to check with the design team how to simplify the design rule of the modal :)

Reply:
completely agree with you!